src/modules/iiif-viewer/IiifViewer.tsx

```diff
@@ -370,7 +373,7 @@ const IiifViewer = forwardRef<IiifViewerFunctions, IiifViewerProps>(
 							</button>
 						);
 					})}
-				</div>

```

Comment:
Add comment why a custom scrollbar is used

Reply:
ok, i've added:
			// Use custom scrollbar because on windows the default scrollbar is big and ugly
			// and here it shows up on the right side of the reference strip, so it extra noticeable